templates/toil-cluster-role-and-policy.yaml

```diff
@@ -0,0 +1,89 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  ClusterName:
+    Description: Name of the cluster that will use these resources.
+    Type: String
+Resources:
+  ClusterRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub "${ClusterName}-role"
```

Comment:
if it's not required you shouldn't really name anything yourself in CFN.  just let CFN handle naming for you.  it will be easier to maintain.